src/core/markdown/MarkdownParser.ts

```diff
@@ -51,16 +51,16 @@ export class MarkdownParser implements Parser {
         try {
             this.stack = [{type: this.schema.topNodeType, content: []}];
 
-            let yfmTokens;
+            let mdItTokens;
             try {
-                yfmTokens = this.tokenizer.parse(text, {});
+                mdItTokens = this.tokenizer.parse(text, {});
             } catch (err) {
                 const e = err as Error;
                 e.message = 'Unable to parse your markup. Please check for errors. ' + e.message;
                 throw e;
             }
 
-            this.parseTokens(yfmTokens);
+            this.parseTokens(mdItTokens);
```

Comment:
nit:
What does `It` in `mdItTokens` mean?

Reply:
`mdIt` is short of `markdown-it`